1967.number-of-strings-that-appear-as-substrings-in-word.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/number-of-strings-that-appear-as-substrings-in-word/
+ */
+
+/**
+ * @param {string[]} patterns
+ * @param {string} word
+ * @return {number}
+ */
+var numOfStrings = function(patterns, word) {
+    
+    let counter = 0;
+    
+    for (let i = 0; i < patterns.length; i++) {
+        if(word.includes(patterns[i]) === true) {
```

Comment:
You can remove the last part of the condition like this.
`if(word.includes(patterns[i])) {`

Reply:
I corrected this code.